crates/signer/src/wallet/private_key.rs

```diff
@@ -14,6 +14,19 @@ use std::str::FromStr;
 use {elliptic_curve::rand_core, std::path::Path};
 
 impl Wallet<SigningKey> {
+    /// Creates a new Wallet instance from a [`SigningKey`].
+    ///
+    /// This can also be used to create a Wallet from a [`SecretKey`](K256SecretKey).
+    /// See also the `From` implementations.
+    #[doc(alias = "from_private_key")]
+    #[doc(alias = "new_private_key")]
+    #[doc(alias = "new_pk")]
+    #[inline]
+    pub fn from_signing_key(signer: SigningKey) -> Self {
```

Comment:
Replace with `From<SigningKey>` and remove?

Reply:
follow up imo